nautobot/extras/models/customfields.py

```diff
@@ -112,8 +112,8 @@ def render(self, context):
             logger.warning("Failed to render computed field %s: %s", self.key, exc)
             return self.fallback_value
 
-    def clean(self):
-        super().clean()
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
```

Comment:
similar change needed/desired for relationships and custom fields?

Also, probably want to reverse the order here - it makes sense to call super().clean() first, but super.save() should only be called *after* we do our own logic, I think?